--- a/packages/nano/build.sh
+++ b/packages/nano/build.sh
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="Small, free and friendly text editor"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="8.7"
-TERMUX_PKG_SRCURL=https://nano-editor.org/dist/latest/nano-$TERMUX_PKG_VERSION.tar.xz
+TERMUX_PKG_SRCURL=https://www.nano-editor.org/dist/v8/nano-$TERMUX_PKG_VERSION.tar.xz
 TERMUX_PKG_SHA256=afd287aa672c48b8e1a93fdb6c6588453d527510d966822b687f2835f0d986e9
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libandroid-support, ncurses"
