--- a/packages/nano/build.sh
+++ b/packages/nano/build.sh
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="Small, free and friendly text editor"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="9.0"
-TERMUX_PKG_SRCURL=https://nano-editor.org/dist/latest/nano-$TERMUX_PKG_VERSION.tar.xz
+TERMUX_PKG_SRCURL=https://www.nano-editor.org/dist/v9/nano-$TERMUX_PKG_VERSION.tar.xz
 TERMUX_PKG_SHA256=9f384374b496110a25b73ad5a5febb384783c6e3188b37063f677ac908013fde
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libandroid-support, ncurses"
